src/main/java/by/exadel/internship/auditing/Auditable.java

```diff
@@ -0,0 +1,39 @@
+package by.exadel.internship.auditing;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.data.annotation.CreatedBy;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import java.time.LocalDate;
+
+@Getter(AccessLevel.PROTECTED)
```

Comment:
Why protected?

Reply:
To use getters or setters only with the inherited classes, for instance Internship extends Auditable<String>

Reply:
deleted and put @Data